### DISCUSSION TITLE
How to set up Editor Js in React-Native?

### DISCUSSION THREAD

**Discussion by ayush-pradhan** [codex-team/editor.js#1806] [Q&A]
I have used **Editor-JS in ReactJS** and simple JS too. In react I have used editor-js in ReactJS and trying to use the same in React-Native but it is giving a reference error:

**Error: Requiring module "node_modules\editor-js\dist\editor-js.umd.js", which threw an exception: ReferenceError: Can't find variable: Element**

I am new to Native and not able to resolve this error.

Code:

`import EditorJS from 'editor-js';`

Initializing Editor

`<div id="editorjs"></div>`

and _**id is not supported in React-Native**_. I can't use any other editor because whole of my app should be consistent with the Web app where EditorJS is being used.

**Comment by hsnfirdaus** [comment]
Well, this is because currently editorjs is just native web javascript application, not react native component...
So if you want to use editorjs in react native, wait for port editorjs of react native or use webview to run html and javascript code on your mobile app...

**Comment by Hidekih** [comment]
I current working at a library to solve this problem. If you want to test it, you can found at: https://www.npmjs.com/package/editorjs-viewer-native
